fr/02_1_Installation_manuelle_d_un_VPS_Bitcoin-Core.md

```diff
@@ -0,0 +1,460 @@
+
+_Si vous avez déjà vos propres techniques pour configurer une machine, allez-y et suivez-les, puis passez à "Configuration d'un utilisateur", puis "Installation de Bitcoin". Sinon, continuez !_
+
+### Configurez votre nom d'hôte
+
+Choisissez un nom d'hôte (_hostname_) pour votre machine (par exemple "mybtc") et entrez-le dans votre fichier de nom d'hôte ; ensuite, exécutez quelques commandes pour propager cette nouvelle information :
+
+```
+$ echo "mybtc" > /etc/hostname
+$ /etc/init.d/hostname.sh start
+$ /bin/hostname "mybtc"
+```
+
+Saisissez les informations relatives à l'adresse IP et au nom d'hôte dans votre fichier /etc/hosts. Notez que vous devez également saisir un nom d'hôte complet, comme indiqué ci-dessous. Si vous n'intégrez pas la machine dans un domaine, il suffit de choisir un suffixe ".local".
+
+```
+$ echo "127.0.0.1    localhost" > /etc/hosts
+$ echo "127.0.1.1 mybtc.local mybtc" >> /etc/hosts
+```
+
+### Mettez à jour votre fuseau horaire
+
+Assurez-vous que votre fuseau horaire est correct.
+
+L'exemple suivant règle votre machine sur le fuseau horaire de la côte ouest américaine :
+
+```
+$ echo "America/Los_Angeles" > /etc/timezone
+$ cp /usr/share/zoneinfo/America/Los_Angeles /etc/localtime
+```
+
+## Protégez votre VPS
+
+Bien que vous n'accordiez pas beaucoup de valeur réelle à ce serveur, vous devez néanmoins vous assurer qu'il est aussi absolument sécurisé que possible.
+
+### Créez des règles de pare-feu
+
+Pour commencer, créez un fichier de règles de pare-feu (_firewall_).
+
+_Pour toutes les instructions qui ressemblent à celle-ci, vous devez simplement pouvoir couper du "cat" jusqu'à l'EOF, et tout sera placé dans le fichier approprié._
+
+```
+$ cat > /etc/iptables.firewall.rules <<EOF
+*filter
+
+#  Allow all loopback (lo0) traffic and drop all traffic to 127/8 that doesn't use lo0
+-A INPUT -i lo -j ACCEPT
+-A INPUT -d 127.0.0.0/8 -j REJECT
+
+#  Accept all established inbound connections
+-A INPUT -m state --state ESTABLISHED,RELATED -j ACCEPT
+
+#  Allow all outbound traffic - you can modify this to only allow certain traffic
+-A OUTPUT -j ACCEPT
+
+# If you want HTTP and HTTPS, uncomment these
+
+#  Allow SSH connections
+#
+#  The -dport number should be the same port number you set in sshd_config
+#
+-A INPUT -p tcp -m state --state NEW --dport 22 -j ACCEPT
+
+#  Allow ping
+-A INPUT -p icmp -j ACCEPT
+
+# Allow Bitcoin connections
+-A INPUT -p tcp --dport 22 -m conntrack --ctstate NEW,ESTABLISHED -j ACCEPT
+-A INPUT -p tcp --dport 8333 -j ACCEPT
+-A INPUT -p tcp --dport 18333 -j ACCEPT
+-A INPUT -p tcp --dport 9735 -j ACCEPT
+-A INPUT -m conntrack --ctstate ESTABLISHED -j ACCEPT
+
+#  Log iptables denied calls
+-A INPUT -m limit --limit 5/min -j LOG --log-prefix "iptables denied: " --log-level 7
+
+#  Drop all other inbound - default deny unless explicitly allowed policy
+-A INPUT -j DROP
```

Comment:
```suggestion
Bien que vous n'accordiez pas beaucoup de valeur réelle à ce serveur, vous devez néanmoins vous assurer de sa sécurité aussi soigneusement que possible.

### Créez des règles de pare-feu

Pour commencer, créez un fichier de règles pare-feu (_firewall_).

_Pour toutes les instructions qui ressemblent à celle-ci, vous devez simplement pouvoir couper du "cat" jusqu'à l'EOF, et tout sera placé dans le fichier approprié._

```
$ cat > /etc/iptables.firewall.rules <<EOF
*filter

#  Allow all loopback (lo0) traffic and drop all traffic to 127/8 that doesn't use lo0
-A INPUT -i lo -j ACCEPT
-A INPUT -d 127.0.0.0/8 -j REJECT

#  Accept all established inbound connections
-A INPUT -m state --state ESTABLISHED,RELATED -j ACCEPT

#  Allow all outbound traffic - you can modify this to only allow certain traffic
-A OUTPUT -j ACCEPT

# If you want HTTP and HTTPS, uncomment these

#  Allow SSH connections
#
#  The -dport number should be the same port number you set in sshd_config
#
-A INPUT -p tcp -m state --state NEW --dport 22 -j ACCEPT

#  Allow ping
-A INPUT -p icmp -j ACCEPT

# Allow Bitcoin connections
-A INPUT -p tcp --dport 22 -m conntrack --ctstate NEW,ESTABLISHED -j ACCEPT
-A INPUT -p tcp --dport 8333 -j ACCEPT
-A INPUT -p tcp --dport 18333 -j ACCEPT
-A INPUT -p tcp --dport 9735 -j ACCEPT
-A INPUT -m conntrack --ctstate ESTABLISHED -j ACCEPT

#  Log iptables denied calls
-A INPUT -m limit --limit 5/min -j LOG --log-prefix "iptables denied: " --log-level 7

#  Drop all other inbound - default deny unless explicitly allowed policy
-A INPUT -j DROP
```